src/components/Controls/Input/styles.scss

```diff
@@ -3,11 +3,13 @@
 .input {
   display: flex;
   align-items: center;
+  width: 100%;
```

Comment:
Doing this you're saying - "All inputs in the app should take the full width of the parent container". I don't think we really want this behavior. What you can do instead is to introduce `fullWidth` property on input that will conditionally add this styling